src/rovr_control/rovr_control/main_control_node.py

```diff
@@ -387,16 +353,22 @@ def joystick_callback(self, msg: Joy) -> None:
 
         # Check if the Apriltag calibration button is pressed
         if msg.buttons[START_BUTTON] == 1 and buttons[START_BUTTON] == 0:
+
             # Start the field calibration process
-            if self.apriltag_timer.is_canceled():
-                self.started_calibration = False
-                self.field_calibrated = False
+            if self.field_calibrated.done():
+                self.get_logger().info("Field calibration button pressed")
+                if not self.act_calibrate_field_coordinates.wait_for_server(timeout_sec=1.0):
```

Comment:
self.act_calibrate_field_coordinates does not seem to be defined anywhere, which causes the node to crash